Makefile

```diff
@@ -59,7 +59,7 @@ TEST_PREREQUISITES=${INSTALL_TARGET}
 RUN_PREREQUISITES=${INSTANCE_TARGET}
 
 ##############################################################################
-# CONVINIENCE
+# CONVENIENCE
```

Comment:
@petschki as this file is auto-generated, maybe these typos here can be upstreamed? 🤔